x-pack/platform/plugins/shared/alerting/common/alert_schema/field_maps/mapping_from_field_map.test.ts

```diff
@@ -219,6 +219,9 @@ describe('mappingFromFieldMap', () => {
                 action_group: {
                   type: 'keyword',
                 },
+                attack_ids: {
```

Comment:
Since we decided to move this field to security and attack discovery alerts, we do not need this definition here anymore.